src/components/AnonymousUsername/index.js

```diff
@@ -45,7 +45,7 @@ const AnonymousUsername = (props) => {
     );
   }
 
-  return <Text style={[styles.feedUsername, props.style]}>Anonymous</Text>;
+  return <Text style={[styles.feedUsername(isFeed), props.style]}>Loading...</Text>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from displaying "Anonymous" to "Loading..." might be confusing for users if the loading process takes longer than expected. Consider adding a timeout or some other form of error handling to revert back to "Anonymous" if the loading process exceeds a certain duration.
```diff
- return <Text style={[styles.feedUsername(isFeed), props.style]}>Loading...</Text>;
+ const [isLoading, setIsLoading] = useState(true);
+
+ useEffect(() => {
+   const timer = setTimeout(() => {
+     setIsLoading(false);
+   }, 5000); // 5 seconds timeout
+
+   return () => clearTimeout(timer);
+ }, []);
+
+ return <Text style={[styles.feedUsername(isFeed), props.style]}>{isLoading ? 'Loading...' : 'Anonymous'}</Text>;
```
This way, the text will revert back to "Anonymous" after 5 seconds if it's still in the loading state.


<!-- This is an auto-generated comment by OSS CodeRabbit -->